app/_gateway_entities/consumer.md

```diff
@@ -143,7 +145,19 @@ Common use cases for Consumers:
 
 ## Centrally-managed Consumers 
 
-<!-- @todo: migrate from Konnect docs: https://docs.konghq.com/konnect/centralized-consumer-management/ -->
+Consumers can be scoped to a {{site.konnect_short_name}} region and managed centrally, or be scoped to a Control Plane in Gateway Manager.
+
+Centralized Consumer management provides the following benefits:
+* Set up a Consumer identity centrally instead of defining it in multiple Control Planes.
+* Share Consumers across multiple Control Planes. Users don't need to replicate changes to Consumer identity in multiple Control Planes and Consumer configuration doesn't conflict.
```

Comment:
```suggestion
* **Avoid conflicts from duplicate Consumer configuration**: Users don't need to replicate changes to Consumer identity in multiple Control Planes and Consumer configuration doesn't conflict.
```
The previous "title" made it sound like basically the same thing as the first bullet point.